src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
 // write code here
+const table = document.querySelector('tbody');
+const rows = document.getElementsByTagName('tr');
+
+const appendRowButton = document.querySelector('.append-row');
+const deleteRowButton = document.querySelector('.remove-row');
+const appendColumnButton = document.querySelector('.append-column');
+const deleteColumnButton = document.querySelector('.remove-column');
+
+const growthTable = () => {
+  appendRowButton.addEventListener('click', () => {
+    const lastRow = rows[rows.length - 1];
+    const cloneRows = lastRow.cloneNode(true);
```

Comment:
If the table is initially empty, accessing `rows[rows.length - 1]` will result in an error. Consider adding a check to ensure there is at least one row before attempting to clone it.